src/main/steps/common/form/contactForHelp.njk

```diff
@@ -0,0 +1,24 @@
+<article>
+
+<details class="govuk-details" data-module="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      Contact us for help
```

Comment:
These texts does not support welsh, can you change the logic and move it to `common.content.ts` file?